### ISSUE TITLE
Improve article 1608.05377

### ISSUE THREAD

**Issue by JessRiedel** [dginev/ar5iv#8]
**Exact location of issue**

https://ar5iv.org/html/1608.05377#S0.F1

**Problem details**

Figure 1, a vector graphic embedded in the original PDF, has been converted incorrectly to an SVG image in the HTML5 version of the paper.  
- All the typeset math characters have been removed
- The smooth curve shapes have been replaced by rectangles with rounded corners
- Many colors changed.
- Shading removed

Figure 1 as it appears in the original PDF:

<img width="526" alt="Screen Shot 2022-01-31 at 1 03 42 PM" src="https://user-images.githubusercontent.com/16365564/151875753-4af1f5c1-1f12-425c-a22c-4a3ace7a4cd3.png">

Figure 1 as rendered on ar5iv.org:

<img width="808" alt="Screen Shot 2022-01-31 at 1 04 01 PM" src="https://user-images.githubusercontent.com/16365564/151875776-c99dac6c-7cae-403d-85e4-1d215e8d8140.png">

**Suggested resolution**
Although vector images are generally preferable, in my opinion the image converter should be able to tell if it's encountering elements it can't handle and fall back to a raster conversion.  Lightweight graphics are nice, but silently corrupting images is very bad.

**Desktop**
 - OS: MacOS Catalina
 - Browser: Chrome

**Smartphone:**
 - Device: iPhone 12
 - OS: iOS 15.2.1
 - Browser: Chrome

**Additional context**
This image was originally created in Keynote on MacOS.  (I am the author.)

**Comment by JessRiedel** [comment]
Oh, holy cow, I just realized this is the old version of the image that is part of the "v1" version of the paper on the arXiv rather than the current "v2".  It looks like ar5iv.org doesn't have a way to choose version number. ( https://ar5iv.org/html/1608.05377v2 has the same problem.)

I just checked the text and confirmed that ar5iv. org is serving the complete old version of the paper (not just the figure).  Sorry for misdiagnosing this.
